src/apiGithub.js

```diff
@@ -0,0 +1,82 @@
+import got from 'got';
+import { debugApiGithub } from './config.js';
+
+export default class ApiGithub {
+    constructor (token) {
```

Comment:
This class has public and private methods as well. Generally we try to cover public API of modules with unit tests. Let's write 'em then! I also suggest to use `proxyquire` and `sinon` to mock and stub the `gotz module and prevent actual HTTP requests from being made.